app/views/posts/index.html.erb

```diff
@@ -1 +1,26 @@
-<h1>This page will show a Single User's posts and show posts comments</h1>
\ No newline at end of file
+<div class=box>
+    <%= image_tag @user.photo, class: 'img' %>
+    <div class=user-box>
+        <%= @user.name %><br>
+        number of posts: <%= @user.posts_counter %><br>
+    </div>
+</div>
+
+<% @posts_all.each_with_index do |post, idx| %>
+    <%= link_to user_post_path(user_id: @user.id, id: post.id) do %>
+    <div class=rectangle>
+        Post #<%= idx+1 %><br><br>
+        <%= post.text %><br><br>
+        comments: <%= post.comments_counter %>
+        likes: <%= post.likes_counter %><br>
+    </div>
+    <% end %>
+
+    <% if post.comments.length != 0 %>
+    <div class=rectangle>
+        <% post.most_recent_comments.each do |comment| %>
+        <%= comment.author.name %>: <%= comment.text %><br><br>
+        <% end %>
+    </div>
+    <% end %>
```

Comment:
- [ ] Kindly follow the given design as this is a core requirement

Your design
![image](https://user-images.githubusercontent.com/81932478/204848940-6ca81329-cb42-4c8b-9b59-0ef6818c47aa.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/204849067-5bd52a7c-1a1f-4886-8238-a554d47c4fa7.png)

- Kindly move the number of posts to the left of the design as it is in the given design
- Also move the comments and likes counters to the left as it is in the given design

Reply:
@Munsa1 Thank you for your review of this PR.
I have implemented the changes required.

![image](https://user-images.githubusercontent.com/40719899/204870772-998b492e-f0a9-4e5e-a048-af9c38347d64.png)